app/views/users/sign_in.html.haml

```diff
@@ -18,3 +18,50 @@
         %fieldset.form-group.action
```

Comment:
You are correct Brandon. I have been meaning to remove it. Will get my skates on and do it :-)